ferrocene/doc/user-manual/src/rustc/install.rst

```diff
@@ -21,11 +21,15 @@ section of the target documentation.
 Installation
 ------------
 
-.. note::
+Preferred way of installing and managing Ferrocene toolchain(s) is via our
+``rustup`` like tool called ``criticalup``.
+
+Find more details about using ``criticalup`` in the
+`CriticalUp User Documentation <https://criticalup.ferrocene.dev/index.html>`_.
 
-   We are developing a tool similar to ``rustup``, able to manage your
-   Ferrocene installations and install Ferrocene without relying on a web
-   browser. It will be available soon.
+.. note::
+    Are you trying to target a specific architecture using cross-compilation?
```

Comment:
The UM is not currently interrogative. Perhaps consider an instructive tone.

In the end, this feels redundant as immediately above it we instruct them of basically the same thing: https://docs-dev.ferrocene.dev/nightly/user-manual/rustc/install.html#prerequisites